builds/e2e/templates/stresstest-deploy-windows.yaml

```diff
@@ -15,17 +15,11 @@ parameters:
   mqtt.settings.enabled: 'true'
   loadGen.message.frequency: ''
   loadGen1.transportType: 'Amqp_Tcp_Only'
-  loadGen2.transportType: 'Amqp_WebSocket_Only'
+  loadGen2.transportType: 'Amqp_Tcp_Only'
   loadGen3.transportType: 'Mqtt_Tcp_Only'
-  loadGen4.transportType: 'Mqtt_WebSocket_Only'
+  loadGen4.transportType: 'Mqtt_Tcp_Only'
 
 steps:
-  - task: CopyFiles@2
```

Comment:
just add a simple comment here why remove "copy edgelet artifact"; same for other template files.